Makefile.inc

```diff
@@ -13,14 +13,15 @@
 # NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE
 # USE OR PERFORMANCE OF THIS SOFTWARE.
 #
-CC = gcc
+CC ?= gcc
```

Comment:
You can remove it. 
It allows to take default host tool-chain and compatible with cross-compiler as well.